docs/faq/network/blockchain-architecture.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_label: 'Blockchain Architecture'
+sidebar_position: 1
+---
+
+# Blockchain Architecture
+
+### What is LUKSOs Network Architecture?
+
+LUKSO's network architecture is based on the **Proof of Stake** consensus protocol, which relies on validators to secure the network and maintain consensus. The protocol is combined with two fundamental necessities to allow a secure and efficient network.
+
+The first necessity is **achieving the finality** of changes made to the network. This ensures that once a block is added to the blockchain, its information is immutable and cannot be changed. Finality is managed by the Casper the Friendly Finality Gadget (**Casper FFG**), a consensus mechanism designed to achieve this finality.
+
+The second necessity is having a defined **ruleset to determine block arrangement**. This is particularly crucial when multiple blocks require simultaneous validation. In such cases, disputes can lead to splits of the linear chain at the latest common block, requiring validators to choose which of them they want to validate. LUKSO tackles this problem with the Latest Message Driven Greediest Heaviest Observed Subtree (**LMD GHOST**) protocol. The algorithm provides the additional ruleset necessary to ensure unified computing on one branch, thereby resolving the problem of blockchain splits.
+
+### What is Proof of Stake?
+
+PoS is a consensus mechanism many blockchain networks use to **validate and add new transactions**. Unlike Proof of Work, which requires miners to solve complex mathematical problems to add new blocks to the blockchain, PoS relies on the amount of cryptocurrency a person holds and is willing to stake as collateral.
+
+In a PoS blockchain, validators are chosen to create a new block based on their **stake in the network**. The more cryptocurrency a person holds, the chances of being selected as a validator increase. Once chosen, validators validate transactions and add them to the blockchain. For their service, validators are rewarded with transaction fees and potentially new coins.
```

Comment:
replace "cryptocurrency" by "native tokens"

Reply:
also, this "The more cryptocurrency a person holds, the chances of being selected as a validator increase" i'm not sure:
- if you mean that with more native token you can create more validators, thus having more chances to validate blocks, ok, but it needs to be rephrased
- if you mean having more than 32LYX will give you more chances to be selected, then it's not true